lib/point_quest/authentication/actor.ex

```diff
@@ -46,4 +46,8 @@ defmodule PointQuest.Authentication.Actor do
   end
 
   @type t :: Adventurer.t() | PartyLeader.t()
+
+  def get_actor_id(%PartyLeader{adventurer: %{id: id}}), do: id
+  def get_actor_id(%PartyLeader{leader_id: id}), do: id
+  def get_actor_id(%Adventurer{adventurer: %{id: id}}), do: id
```

Comment:
hrmmm this felt good in the moment, but on review I don't like that I'm lying. In 2 of the 3 cases here, I'm actually returning the actor's adventurer ID...

Reply:
I think it's fine for now. I do wonder if the party leaders adventurer model could need some changes to alleviate some of the funkiness we are experiencing